packages/daimo-common/test/daimoLink.test.ts

```diff
@@ -81,27 +89,27 @@ test("DaimoLink from daimo:// direct app link", () => {
   const url = "daimo://account/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93";
   assert.deepStrictEqual(parseDaimoLink(url), {
     type: "account",
-    addr: "0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93",
+    account: "0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93",
   });
 });
 
 test("DaimoLink normalization", () => {
   for (const [url, link] of testCases) {
     // Ensure addresses always end up checksummed
     assert.deepStrictEqual(parseDaimoLink(url.toLowerCase()), link);
+  }
 
-    // Ensure that amount is normalized
-    const variants = [
-      "https://example.com/link/request/123/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93/1.00001",
-      "https://example.com/link/request/123/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93/1.0",
-      "https://example.com/link/request/123/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93/1",
-    ];
-    const correct =
-      "https://example.com/link/request/123/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93/1.00";
+  // Ensure that amount is normalized
+  const variants = [
+    "https://example.com/link/request/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93/1.00001/123",
+    "https://example.com/link/request/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93/1.0/123",
+    "https://example.com/link/request/0x061b0a794945fe0Ff4b764bfB926317f3cFc8b93/1/123",
```

Comment:
i'd add an example of the ENS or daimo name

Reply:
done in test cases above